src/modules/reserve.js

```diff
@@ -0,0 +1,42 @@
+import setPage from './setpage';
+import makeForm from './form';
+
+function reserve() {
+  const container = setPage('reserve-link');
+  container.setAttribute('class', 'container reserve');
+  const row = document.createElement('div');
+  row.setAttribute('class', 'row');
+
+  const colOne = document.createElement('div');
+  colOne.setAttribute('class', 'd-none d-lg-block d-xl-block col-lg-7 image');
+  colOne.style.backgroundImage = "url('./img/tables.jpg')";
```

Comment:
Not sure why are you doing this here and not on your `scss`.

Reply:
I did not want to webpack images.